package.json

```diff
@@ -6,7 +6,7 @@
     "start": "make start",
     "stop": "make stop",
     "test": "make test",
-    "smoke": "tap ./test/integration/smoke-testing/*.js --timeout=3600",
+    "smoke": "tap ./test/integration/smoke-testing/*.js --timeout=3600 --no-coverage -R classic",
```

Comment:
We should probably also add `--no-coverage -R classic` to all the other tap tests to keep the output similar across them.

Reply:
I added it to smoke-sauce, but I only added the --no-coverage to smoke-verbose since it already used a different report method, -R spec